index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel = "stylesheet" type = "text/css" href="../css/main.css">
```

Comment:
u don't have to navigate using .., check about  **base** tag, u should be using absolute path